controllers/front/CartController.php

```diff
@@ -653,15 +673,35 @@ private function checkCartProductsMinimalQuantities()
         foreach ($productList as $product) {
             if ($product['minimal_quantity'] > $product['cart_quantity']) {
                 // display minimal quantity warning error message
-                $this->errors[] = $this->trans(
-                    'The minimum purchase order quantity for the product %product% is %quantity%.',
-                    [
-                        '%product%' => $product['name'],
-                        '%quantity%' => $product['minimal_quantity'],
-                    ],
-                    'Shop.Notifications.Error'
+                $this->errors[] = $this->translateProductNameWithAttributes(
+                    'The minimum order quantity for the product [1]%product%[/1] is [1]%quantity%[/1].',
```

Comment:
```suggestion
                    'Select at least [1]%quantity%[/1] items for the product [1]%product%[/1].',
```

Reply:
Can the minimum order quantity be **1**? Just so we know if we might need a singular and a plural string.

Reply:
The minimal quantity is always 1 by default, but that check is always false until you set the minimal quantity 2 or more 😊 
To be short, no singular string/term needed.

About the suggestion, we should not strictly refer to product as _items_ because they could be virtual products or whatever, I mean, for me _items_ is something physical.
wdyt?

Reply:
I believe an _item_ can both refer to a virtual or physical object, but in this context, I think you're right — product is more fitting.